person.rb

```diff
@@ -18,4 +23,17 @@ def can_use_services?
   def of_age?
     age >= 18
   end
+
+  public
+
+  def correct_name
+    @name
+  end
 end
+
+person = Person.new(22, 'maximilianus')
+p person.correct_name
+capitalized_person = capitalize_decorator.new(person)
```

Comment:
OPTIONAL
- [ ] To test your app correctly change `capitalize_decorator` to CapitalizeDecorator

Reply:
@rloterh It seems linters does not allow that kind of naming 
![Capture](https://user-images.githubusercontent.com/61447440/180058555-7924ee04-569d-4608-9366-f95f3eaf3ab2.PNG)